<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>dlgNewProfileCreate</class>
 <widget class="QDialog" name="dlgNewProfileCreate">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1070</width>
    <height>834</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QHBoxLayout" name="horizontalLayout_16">
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_12">
     <item>
      <layout class="QVBoxLayout" name="verticalLayout_3">
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout">
         <item>
          <widget class="QLabel" name="label">
           <property name="text">
            <string>Profile Name:</string>
           </property>
           <property name="buddy">
            <cstring>lineEdit_ProfileName</cstring>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_ProfileName"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="horizontalLayout_6">
         <item>
          <widget class="QLabel" name="label_11">
           <property name="text">
            <string>Copy Profile Name:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QLineEdit" name="lineEdit_copyProfileName"/>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout_5">
       <property name="sizeConstraint">
        <enum>QLayout::SetDefaultConstraint</enum>
       </property>
       <item>
        <widget class="QTabWidget" name="tabWidget">
         <property name="sizePolicy">
          <sizepolicy hsizetype="Fixed" vsizetype="Expanding">
           <horstretch>0</horstretch>
           <verstretch>0</verstretch>
          </sizepolicy>
         </property>
         <property name="currentIndex">
          <number>2</number>
         </property>
         <widget class="QWidget" name="tab_sourceDSN">
          <attribute name="title">
           <string>Source</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <item>
            <widget class="QTextEdit" name="textEdit_sourceDSN"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_targetDSN">
          <attribute name="title">
           <string>Target</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_3">
           <item>
            <widget class="QTextEdit" name="textEdit_targetDSN"/>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_Properties">
          <attribute name="title">
           <string>Properties</string>
          </attribute>
          <layout class="QVBoxLayout" name="verticalLayout_6">
           <item>
            <widget class="QGroupBox" name="groupBox_4">
             <property name="title">
              <string>Profile Properties</string>
             </property>
             <layout class="QVBoxLayout" name="verticalLayout_7">
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_13">
                <item>
                 <widget class="QLabel" name="label_8">
                  <property name="text">
                   <string>Description:</string>
                  </property>
                  <property name="alignment">
                   <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QTextEdit" name="textEdit_description"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_15">
                <item>
                 <widget class="QLabel" name="label_10">
                  <property name="text">
                   <string>Network Packet Size:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_networkPacketSize"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_14">
                <item>
                 <widget class="QLabel" name="label_9">
                  <property name="text">
                   <string>Commit Throttle:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_commitThrottle"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_7">
                <item>
                 <widget class="QLabel" name="label_2">
                  <property name="text">
                   <string>Drop Target:</string>
                  </property>
                  <property name="buddy">
                   <cstring>comboBox_dropTarget</cstring>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_dropTarget">
                  <item>
                   <property name="text">
                    <string>Drop</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Truncate</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Delete From</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_8">
                <item>
                 <widget class="QLabel" name="label_3">
                  <property name="text">
                   <string>Logging Level:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QComboBox" name="comboBox_loggingLevel">
                  <item>
                   <property name="text">
                    <string>Info</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Debug</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Trace</string>
                   </property>
                  </item>
                  <item>
                   <property name="text">
                    <string>Warn</string>
                   </property>
                  </item>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_9">
                <item>
                 <widget class="QLabel" name="label_7">
                  <property name="text">
                   <string>Profile Enabled:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_profileEnabled">
                  <property name="text">
                   <string>Yes/No</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_10">
                <item>
                 <widget class="QLabel" name="label_6">
                  <property name="text">
                   <string>AutoCommit  Enabled:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QCheckBox" name="checkBox_autoCommitEnabled">
                  <property name="text">
                   <string>Yes/No</string>
                  </property>
                 </widget>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_11">
                <item>
                 <widget class="QLabel" name="label_5">
                  <property name="text">
                   <string>Max Execution Threads:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QSpinBox" name="spinBox_ExecutionThreads"/>
                </item>
               </layout>
              </item>
              <item>
               <layout class="QHBoxLayout" name="horizontalLayout_12">
                <item>
                 <widget class="QLabel" name="label_4">
                  <property name="text">
                   <string>Log File Name:</string>
                  </property>
                 </widget>
                </item>
                <item>
                 <widget class="QLineEdit" name="lineEdit_logFileName"/>
                </item>
               </layout>
              </item>
              <item>
               <spacer name="verticalSpacer_2">
                <property name="orientation">
                 <enum>Qt::Vertical</enum>
                </property>
                <property name="sizeHint" stdset="0">
                 <size>
                  <width>20</width>
                  <height>40</height>
                 </size>
                </property>
               </spacer>
              </item>
             </layout>
            </widget>
           </item>
          </layout>
         </widget>
         <widget class="QWidget" name="tab_tables">
          <attribute name="title">
           <string>Tables</string>
          </attribute>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <item>
            <widget class="QListWidget" name="listWidget_tables"/>
           </item>
          </layout>
         </widget>
        </widget>
       </item>
       <item>
        <widget class="QGroupBox" name="groupBox">
         <property name="title">
          <string>Actions</string>
         </property>
         <layout class="QVBoxLayout" name="verticalLayout">
          <item>
           <widget class="QPushButton" name="pushButton_check">
            <property name="text">
             <string>Check</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_generate">
            <property name="text">
             <string>Generate</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_commit">
            <property name="text">
             <string>Commit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_edit">
            <property name="text">
             <string>Edit</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_delete">
            <property name="text">
             <string>Delete</string>
            </property>
           </widget>
          </item>
          <item>
           <widget class="QPushButton" name="pushButton_copy">
            <property name="text">
             <string>Copy</string>
            </property>
           </widget>
          </item>
          <item>
           <spacer name="verticalSpacer">
            <property name="orientation">
             <enum>Qt::Vertical</enum>
            </property>
            <property name="sizeHint" stdset="0">
             <size>
              <width>20</width>
              <height>40</height>
             </size>
            </property>
           </spacer>
          </item>
         </layout>
        </widget>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item>
    <layout class="QVBoxLayout" name="verticalLayout_5">
     <item>
      <widget class="QGroupBox" name="groupBox_3">
       <property name="layoutDirection">
        <enum>Qt::RightToLeft</enum>
       </property>
       <property name="title">
        <string/>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_4">
        <item>
         <widget class="QDialogButtonBox" name="buttonBox">
          <property name="sizePolicy">
           <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
            <horstretch>0</horstretch>
            <verstretch>0</verstretch>
           </sizepolicy>
          </property>
          <property name="orientation">
           <enum>Qt::Vertical</enum>
          </property>
          <property name="standardButtons">
           <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
     <item>
      <widget class="QGroupBox" name="groupBox_2">
       <property name="sizePolicy">
        <sizepolicy hsizetype="Expanding" vsizetype="Preferred">
         <horstretch>0</horstretch>
         <verstretch>0</verstretch>
        </sizepolicy>
       </property>
       <property name="title">
        <string>SQL Statements</string>
       </property>
       <layout class="QVBoxLayout" name="verticalLayout_2">
        <item>
         <widget class="QTabWidget" name="tabWidget_2">
          <property name="currentIndex">
           <number>3</number>
          </property>
          <widget class="QWidget" name="table_fetch">
           <attribute name="title">
            <string>Fetch</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_8">
            <item>
             <widget class="QListWidget" name="listWidget_fetchStatements"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_insert">
           <attribute name="title">
            <string>Insert</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_9">
            <item>
             <widget class="QListWidget" name="listWidget_insertStatements"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_update">
           <attribute name="title">
            <string>Update</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_10">
            <item>
             <widget class="QListWidget" name="listWidget_updateStatements"/>
            </item>
           </layout>
          </widget>
          <widget class="QWidget" name="tab_createstatements">
           <attribute name="title">
            <string>Create</string>
           </attribute>
           <layout class="QVBoxLayout" name="verticalLayout_11">
            <item>
             <widget class="QListWidget" name="listWidget_createStatements"/>
            </item>
           </layout>
          </widget>
         </widget>
        </item>
       </layout>
      </widget>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>dlgNewProfileCreate</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>dlgNewProfileCreate</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
 </connections>
</ui>
